src/types/index.d.ts

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+
+declare type StyleObject = {
+  [key: string]: string | number;
+};
+
+export type DataEntry = {
```

Comment:
Not an issue - just my opinion - should the name of this be more specific to the library ie. `PieItem`. Then the import on other projects would be `import PieChart, { PieItem } from 'react-minimal-pie-chart';`.

Reply:
I'll go with `PieChartData` and `PieChartExtendedData`, then.